setup.cfg

```diff
@@ -22,9 +23,99 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
+[metadata]
+name = invenio-github
+version = attr: invenio_github.__version__
+description = "Invenio module that adds GitHub integration to the platform."
+long_description = file: README.rst, CHANGES.rst
+keywords = invenio github
+license = GPLv2
+author = CERN
+author_email = info@inveniosoftware.org
+platforms = any
+url = https://github.com/inveniosoftware/invenio-github
+classifiers =
+    Development Status :: 5 - Production/Stable
 
-[aliases]
-test=pytest
+[options]
+packages = find:
+zip_safe = False
+include_package_data = True
+python_requires = >= 3.7
+install_requires =
+    invenio-i18n>=2.0.0
+    Flask-Breadcrumbs>=0.4.0
+    Flask-Menu>=0.5.0
+    Flask>=1.0.4
+    email-validator>=1.0.5
+    github3.py>=4.0.1,<5.0.0
+    humanize>=0.5.1
+    invenio-assets>=2.0.0
+    invenio-accounts>=2.2.0
+    invenio-celery>=1.2.5
+    invenio-db>=1.1.2
+    invenio-formatter>=1.0.1
+    invenio-oauth2server>=2.0.0
+    invenio-oauthclient>=2.3.0
+    invenio-pidstore>=1.2.1,<2.0.0
+    invenio-records-rest>=2.2.0
+    invenio-rest>=1.2.4  # TODO needs a release. current: 1.2.8
+    invenio-webhooks>=1.0.0a4  # TODO invenio webhooks need a release after being updated (major 2.0.0)
```

Comment:
`invenio-webhooks` needs to be released (major, `2.0.0`) and we need to bump it here.